src/deploy/D3MInit.sol

```diff
@@ -187,16 +200,17 @@ library D3MInit {
         );
         dss.spotter.poke(ilk);
 
+        GemAbstract gem = GemAbstract(ID3MPool(d3m.pool).redeemable());
```

Comment:
Require this is non zero? From PR#93 (same discussion) but we could require gem.decimals() and symbol are also not 0? 

Up to you

Reply:
Personally I think it's okay as if the address is 0 or really any non-erc20-like contract it will revert when attempting to get the decimals().